doc/release-notes/release-8.sgml.in

```diff
@@ -102,14 +101,14 @@ This section gives an account of those changes in three categories:
 <sect1>Changes to existing options<label id="modifiedoptions">
 <p>
 <descrip>
-	<tag>--enable-auth-basic=</tag>
-	<p>Removed <em>SMB_LM</em> helper, in favour of the <em>ntlm_auth</em>
-	   alternative offered by the Samba project.
-
 	<tag>--enable-external-acl-helpers=</tag>
 	<p>Removed <em>LM_Group</em> helper. The LM protocol is
 	   insecure and no longer supported on the market since 2008.
 
+	<tag>--enable-auth-ntlm=</tag>
```

Comment:
still broken, should be adding the `--enable-basic-auth` (removed in the PR 2014 not yet committed so cant fix yet)  and nothing said about `ntlm-auth`